pkg/provider/acme/provider.go

```diff
@@ -50,6 +52,8 @@ func (a *Configuration) SetDefaults() {
 	a.CAServer = lego.LEDirectoryProduction
 	a.Storage = "acme.json"
 	a.KeyType = "RSA4096"
+	a.RenewBeforeExpiry = 720
+	a.ExpirationCheckInterval = 1
```

Comment:
Is there a reason to change the default interval from 24 hours to 1 hour? Probably better to maintain the old default if it is overridable?

Reply:
Yeah, you're right. It's better to maintain previous behaviour as default.